docs/openapi-issuer.yaml

```diff
@@ -233,6 +233,36 @@ paths:
         default:
           $ref: "#/components/responses/Unexpected"
 
+  /notification:
```

Comment:
```suggestion
  /notifications:
```

Reply:
What about:

`/signature-request/{id}/notifications` ?

Reply:
I think that `notifications` is not a property of the signature request. I'd rather have separated it

Reply:
@lucacavallaro what about this?

Reply:
1. the issuer can only send **1** `IO Message` per `Signature Request` .
2. the `IO Message IO` must be stored

I propose to add a property called `notification` to `Signature Request` (yes, "_notification_" not "notifications" since the notification is only one) and a related `PUT /signature-requests{id}/notification` to update it (when it's empty)